Integrations/Uptycs/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [19.8.0] - 2019-08-06
 #### New integration Uptycs
```

Comment:
Seems to only appear in uptycs, not sure if should be in other 2 new integrations as well or not here.

Reply:
this was the only one with release notes so i left it...
we never forced release notes for new content entities.